test/core/storages/key_value_store.test.ts

```diff
@@ -228,10 +228,11 @@ describe('KeyValueStore', () => {
             let counter = 0;
 
             for (const char of INVALID_CHARACTERS) {
+                const key = `my_id_${char}`;
```

Comment:
not that it would matter much, but i personally don't like this way to test failures, better use `expect.rejects.toThrow` in general

```ts
const err = `The "key" argument "${key}" must be at most 256 characters`;
await expect(store.setValue(key, 'value')).rejects.toThrow(err);
```

(then you don't need the `counter` at all, since you test that each case fails anyway)